src/templates/resource.js

```diff
@@ -1,57 +1,11 @@
 import React from "react"
```

Comment:
I don't know if or where resource.js is used. that uncertainty should block merging.